pipelines/commands/status_test.go

```diff
@@ -0,0 +1,150 @@
+package commands
+
+import (
+	"github.com/gookit/color"
+	"github.com/jfrog/jfrog-client-go/pipelines/services"
+	"testing"
+	"time"
+)
+
+func Test_monitorStatusChange(t *testing.T) {
```

Comment:
Why use snake case here? It is against this project's conventions.
Please use came case in all tests